docs/COMPILE.md

```diff
@@ -49,6 +49,12 @@ sudo systemctl restart systemd-binfmt
 
 As most RK3399 device run AARCH64 OS, you'll need an `armhf` multiarch environment, and an armhf gcc: On debian, install it with `sudo apt install gcc-arm-linux-gnueabihf`.
 
+#### for Allwinner A64
+
+`mkdir build; cd build; cmake .. -DA64=1 -DCMAKE_BUILD_TYPE=RelWithDebInfo; make -j$(nproc)`
+
+As most Allwinner A64 device run AARCH64 OS, you'll need an `armhf` multiarch environment, and an armhf gcc: On debian, install it with `sudo apt install gcc-arm-linux-gnueabihf`.
```

Comment:
I know this already has been merged, but I don't like that you're calling it "armhf". "armhf" is ambiguous and can either mean ARMv7 or ARMv6. I'm not even sure which you mean here. What are your thoughts?

Reply:
ARMHF is ARMv7 HardFloat
On Debian and Ubuntu, you use `sudo dpkg --add-architecture armhf` to add the armhf architecture (using multiarch).

Reply:
Yes, but on e.g. Alpine ARMHF is ARMv6 Hard Float, and ARMv7 is just called ARMv7. But, sure, more people will probably think of Debian's name for it.

Reply:
I see armhf is just how it's called in other places too, so not really an issue specific to this PR then.

Reply:
There is multiarch available on Alpine?

Reply:
There's not, but they support multiple architectures and you can create an armhf chroot for example. That said, I don't think Box86 builds on musl anyway, so maybe it doesn't matter.

Reply:
Yeah, musl is a problem indeed.